README.md

```diff
@@ -109,7 +109,9 @@ permissions:
 | `commit-sha`     | Used to specify the GitHub commit sha to use                                                            | -       |
 | `ignore-failure` | Used to not fail the gh-action in case of releasability check failure                                   | `false` |
 
-## Versioning
+## Development
```

Comment:
This new input should be documented in the previous table called "Options" - coherence